frontend/cli.py

```diff
@@ -192,6 +192,14 @@ def cli():
     )
 ]
 
+CONFIG_OPTIONS = [
+    click.option(
+        "--config",
```

Comment:
should this be `--config-file` so that the variable name `config` is kept only for a `Config` object (line 376)?

Reply:
Or `--config-yml-file` for specificity